### PR TITLE
To Delete: Palette Virtual Cluster Pause/Resume

### PR DESCRIPTION
**Pull request by resmaa**
*To be Released Feature*. (Draft)
Palette Virtual Cluster Pause/Resume
image

Some links to be updated. Other PR dependencies are there, so can be updated post-merge of dependent Prs.

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/976#" title="Author self-approved">resmaa</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by spectro-prow** [comment]
Hi @resmaa. Thanks for your PR.

I'm waiting for a [spectrocloud](https://github.com/orgs/spectrocloud/people) member to verify that this patch is reasonable to test. If it is, they should reply with `/ok-to-test` on its own line. Until that is done, I will not automatically test new commits in this PR, but the usual testing commands by org members will still work. Regular contributors should [join the org](https://github.com/orgs/spectrocloud/people) to skip this step.

Once the patch is verified, the new status will be reflected by the `ok-to-test` label.

I understand the commands that are listed [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

File: content/docs/04.5-devx/07-sandbox-clusters.md
Comment (karl-cardenas-coding) [inline content/docs/04.5-devx/07-sandbox-clusters.md:54]: The Palette ??? Did you mean the Palette platform?

Comment (karl-cardenas-coding) [reply content/docs/04.5-devx/07-sandbox-clusters.md:54]: Pause and Resume are not nouns, so they should not be in title case.

Comment (karl-cardenas-coding) [reply content/docs/04.5-devx/07-sandbox-clusters.md:54]: Is it only for users with optimized resources? It's a feature for all users that helps them manage resources more efficiently but more importantly, better manage the cost of running clusters.

Comment (karl-cardenas-coding) [reply content/docs/04.5-devx/07-sandbox-clusters.md:54]: Palette Virtual Clusters should be in title case.

File: content/docs/04.5-devx/07-sandbox-clusters.md
Comment (karl-cardenas-coding) [inline content/docs/04.5-devx/07-sandbox-clusters.md:72]:
```suggestion
* New apps cannot be deployed on a virtual cluster in the paused state.
```
I simplified the language.

File: content/docs/04.5-devx/07-sandbox-clusters.md
Comment (karl-cardenas-coding) [inline content/docs/04.5-devx/07-sandbox-clusters.md:58]: This title is confusing. The content below sounds more like prerequisites, but it's also mixing general information. The two need to be separated.

File: content/docs/04.5-devx/07-sandbox-clusters.md
Comment (karl-cardenas-coding) [inline content/docs/04.5-devx/07-sandbox-clusters.md:79]: @resmaa take a peek at the [How-to layout standard](https://spectrocloud.atlassian.net/wiki/spaces/DE/pages/1774944263/How-to+Layout).  The current content is close to following the standard, but we need to add a validation step and prerequisites.

File: content/docs/04.5-devx/07-sandbox-clusters.md
Comment (karl-cardenas-coding) [inline content/docs/04.5-devx/07-sandbox-clusters.md:63]: Let's be specific, what CPU and memory is it freed? Is it from the overall user's quota?

File: content/docs/04.5-devx/07-sandbox-clusters.md
Comment (karl-cardenas-coding) [inline content/docs/04.5-devx/07-sandbox-clusters.md:60]: I'm not sure I understand what this means. When I read the sentence, I ask myself, "does this mean that the user's quota is always in effect?" Meaning that regardless of the state of a cluster (paused/active), the requested CPU and memory are still counted towards the overall user's quota limit?

Additionally,  quota and virtual clusters should not be in the title case.

File: content/docs/04.5-devx/07-sandbox-clusters.md
Comment (karl-cardenas-coding) [inline content/docs/04.5-devx/07-sandbox-clusters.md:66]:
```suggestion
* Resources such as CPU, storage, and Load Balancers remain allocated to a virtual cluster regardless of the state.
```
Minor language clean-up

File: content/docs/04.5-devx/07-sandbox-clusters.md
Comment (karl-cardenas-coding) [inline content/docs/04.5-devx/07-sandbox-clusters.md:69]:
```suggestion
* The Apps deployed on a virtual cluster go to a pause state when the cluster is paused.
```
The term virtual cluster has been used previously, so it's fine to stick with it.

File: content/docs/04.5-devx/07-sandbox-clusters.md
Comment (karl-cardenas-coding) [inline content/docs/04.5-devx/07-sandbox-clusters.md:75]:
```suggestion
* Virtual clusters in a paused state will continue to appear as an entry in the Palette Dev Engine Console. 
```

**Review by karl-cardenas-coding — CHANGES_REQUESTED** [review]
I left some feedback below. The intro paragraph needs some attention and clarification. There is also some awkward language in the text at times that need simplification.  

This is also a How-to, so we need to follow the [How-to layout style](https://spectrocloud.atlassian.net/wiki/spaces/DE/pages/1774944263/How-to+Layout) and place it on its own page, so it's easy for others to find.

File: content/docs/04.5-devx/07-sandbox-clusters.md
Comment (karl-cardenas-coding) [inline content/docs/04.5-devx/07-sandbox-clusters.md:54]:
```suggestion
Palette allows the pause and resume of Palette Virtual Clusters when not in use. This feature enables the users to optimize resource utilization by pausing the virtual clusters not in use. This adds significant flexibility in managing operating costs and resource management for the Palette Virtual Clusters. 
```

File: content/docs/04.5-devx/07-sandbox-clusters/00-pause-restore-virtual-clusters.md
Comment (karl-cardenas-coding) [inline content/docs/04.5-devx/07-sandbox-clusters/00-pause-restore-virtual-clusters.md:19]:
```suggestion
Palette allows the pause and resume of Palette Virtual Clusters when not in use. This feature enables the users to optimize resource utilization by pausing the virtual clusters not in use. This adds significant flexibility in managing operating costs and resource management for the Palette Virtual Clusters. 
```

File: content/docs/04.5-devx/07-sandbox-clusters/00-pause-restore-virtual-clusters.md
Comment (karl-cardenas-coding) [inline content/docs/04.5-devx/07-sandbox-clusters/00-pause-restore-virtual-clusters.md:23]:
```suggestion
* A Running [Palette Virtual Cluster](/devx/sandbox-clusters#createyoursandboxcluster:).
```

File: content/docs/04.5-devx/07-sandbox-clusters/00-pause-restore-virtual-clusters.md
Comment (karl-cardenas-coding) [inline content/docs/04.5-devx/07-sandbox-clusters/00-pause-restore-virtual-clusters.md:38]: Because these are sequential steps, we should use an ordered list with numbers instead of generic bullet points.

File: content/docs/04.5-devx/07-sandbox-clusters.md
Comment (karl-cardenas-coding) [inline content/docs/04.5-devx/07-sandbox-clusters.md:63]:
```suggestion
* The CPU and memory are freed and returned to the Cluster Group upon pausing a Virtual Cluster.
```
Added 'returned to' make it clearer what is happening.

File: content/docs/04.5-devx/07-sandbox-clusters/00-pause-restore-virtual-clusters.md
Comment (karl-cardenas-coding) [inline content/docs/04.5-devx/07-sandbox-clusters/00-pause-restore-virtual-clusters.md:25]: We need a validation section after this. It may sound silly but how can the user validate the cluster is paused?

Comment (karl-cardenas-coding) [inline content/docs/04.5-devx/07-sandbox-clusters/00-pause-restore-virtual-clusters.md:25]:
```suggestion
#  Pause and Resume a Palette Virtual Cluster
```
This shortens the title

File: content/docs/04.5-devx/07-sandbox-clusters/00-pause-restore-virtual-clusters.md
Comment (karl-cardenas-coding) [inline content/docs/04.5-devx/07-sandbox-clusters/00-pause-restore-virtual-clusters.md:19]:
```suggestion
Palette allows the pause and resume of Palette Virtual Clusters when not in use. This feature enables the users to optimize resource utilization by pausing the Virtual Clusters, not in use. This adds significant flexibility in managing operating costs and resource management for the Palette Virtual Clusters. 
```
The intro paragraph is identical to the intro paragraph of the reference page.  This intro paragraph should accomplish two things:
1. Explain to the user that the content below provides instructions on how to pause and restart a cluster.

2. Where to go if they want to learn more about this feature. This would be a link back to the reference page.

File: content/docs/04.5-devx/07-sandbox-clusters.md
Comment (karl-cardenas-coding) [inline content/docs/04.5-devx/07-sandbox-clusters.md:60]: All instances of "Virtual Cluster" should be changed to the virtual cluster. They are all title cased when they should be lower case. We are not using Virtual Clusters as a term after the initial definition of Palette Virtual Cluster. After the initial usage, we can simply refer to the usage as virtual clusters.

File: content/docs/04.5-devx/07-sandbox-clusters.md
Comment (karl-cardenas-coding) [inline content/docs/04.5-devx/07-sandbox-clusters.md:78]: This is great, but we need to explain to the user what the link below is for. It can be as short as a sentence but let's always provide context to the reader. As an example -  "To learn how to pause and resume your Palette Virtual cluster. Check out the how-to guide."

**Review by karl-cardenas-coding — CHANGES_REQUESTED** [review]
It's looking much better. I added more feedback below.

**Review by ritawatson — APPROVED** [review]
Once suggested changes are applied, LGTM.

Questions: 
- Do we use use the term 'invoke' (checked our style guide but didn't see anything about that term.)
- How do we refer to the left navigation pane so I can follow suit. Is it 'left ribbon menu' or 'left navigation pane'?

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/976#" title="Author self-approved">resmaa</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/976#pullrequestreview-1179498671" title="Approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by tiwarisumit3 — APPROVED** [review]

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/976#" title="Author self-approved">resmaa</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/976#pullrequestreview-1179498671" title="Approved">ritawatson</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/976#pullrequestreview-1179508858" title="Approved">tiwarisumit3</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by spectro-prow** [comment]
@resmaa: The following test **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
librarium_build | 16488670e6abe88213b85bcbf0e0a79f49d8fcf5 | [link](http://34.120.121.77/view/gcs/capi-prow-artifacts/pr-logs/pull/spectrocloud_librarium/976/librarium_build/1592470705297952768) | `/test librarium_build`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

File: content/docs/04.5-devx/07-sandbox-clusters.md
Comment (ritawatson) [inline content/docs/04.5-devx/07-sandbox-clusters.md:63]:
```suggestion
* The CPU and memory are freed and returned to the Cluster Group when you pause a virtual cluster.
```

File: content/docs/04.5-devx/07-sandbox-clusters.md
Comment (ritawatson) [inline content/docs/04.5-devx/07-sandbox-clusters.md:66]:
```suggestion
* Resources such as storage, and Load Balancers remain allocated to a virtual cluster regardless of the cluster's state.
```

File: content/docs/04.5-devx/07-sandbox-clusters.md
Comment (ritawatson) [inline content/docs/04.5-devx/07-sandbox-clusters.md:69]:
```suggestion
* The Apps deployed on a virtual cluster go to a pause state when the cluster is paused.
```

File: content/docs/04.5-devx/07-sandbox-clusters.md
Comment (ritawatson) [inline content/docs/04.5-devx/07-sandbox-clusters.md:72]:
```suggestion
* New Apps cannot be deployed on a virtual cluster in the paused state.
```

File: content/docs/04.5-devx/07-sandbox-clusters.md
Comment (ritawatson) [inline content/docs/04.5-devx/07-sandbox-clusters.md:75]:
```suggestion
* Virtual clusters in a paused state will continue to appear as an entry in the Palette Dev Engine Console. 
```

File: content/docs/04.5-devx/07-sandbox-clusters/00-pause-restore-virtual-clusters.md
Comment (ritawatson) [inline content/docs/04.5-devx/07-sandbox-clusters/00-pause-restore-virtual-clusters.md:19]:
```suggestion
To optimize resource utilization, Palette allows you to pause and resume virtual clusters that are not in use. This adds significant flexibility in managing operating costs and resource management for virtual clusters. 
```

File: content/docs/04.5-devx/07-sandbox-clusters/00-pause-restore-virtual-clusters.md
Comment (ritawatson) [inline content/docs/04.5-devx/07-sandbox-clusters/00-pause-restore-virtual-clusters.md:32]:
```suggestion
2. Navigate to the **Main Menu** and select **Palette Virtual Cluster**.
```

File: content/docs/04.5-devx/07-sandbox-clusters/00-pause-restore-virtual-clusters.md
Comment (ritawatson) [inline content/docs/04.5-devx/07-sandbox-clusters/00-pause-restore-virtual-clusters.md:35]:
```suggestion
3. Select the name of the virtual cluster to pause. The cluster details page displays.
```

File: content/docs/04.5-devx/07-sandbox-clusters/00-pause-restore-virtual-clusters.md
Comment (ritawatson) [inline content/docs/04.5-devx/07-sandbox-clusters/00-pause-restore-virtual-clusters.md:38]:
```suggestion
4. Click **Settings** and select the **Pause** option. To resume a paused virtual cluster, select the **Resume** option.
```

File: content/docs/04.5-devx/07-sandbox-clusters/00-pause-restore-virtual-clusters.md
Comment (ritawatson) [inline content/docs/04.5-devx/07-sandbox-clusters/00-pause-restore-virtual-clusters.md:2]:
```suggestion
title: "Pause and Resume Palette Virtual Clusters"
```

File: content/docs/04.5-devx/07-sandbox-clusters/00-pause-restore-virtual-clusters.md
Comment (ritawatson) [inline content/docs/04.5-devx/07-sandbox-clusters/00-pause-restore-virtual-clusters.md:3]: If you change the title, does the metaTitle also need updating to match?

**Review by ritawatson — CHANGES_REQUESTED** [review]
Some suggested changes.

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/976#" title="Author self-approved">resmaa</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/976#pullrequestreview-1179508858" title="Approved">tiwarisumit3</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/04.5-devx/07-sandbox-clusters/00-pause-restore-virtual-clusters.md
Comment (resmaa) [reply content/docs/04.5-devx/07-sandbox-clusters/00-pause-restore-virtual-clusters.md:3]: not mandatory

**Comment by resmaa** [comment]
contents merged  https://github.com/spectrocloud/librarium/pull/967/commits/1efcf93585312251814cd70ac35eff684b1ce364

**Pull request closed without merging**
